src/components/AddItemForm.js

```diff
@@ -69,9 +69,24 @@ class AddItemForm extends React.Component {
         </label>
         <br />
         <input className="submit-btn" type="submit" value="Submit" />
+        <FirestoreCollection
+          path="items"
+          filter={['name', '==', this.state.name]}
+          render={({ data }) => {
+            if (data.length > 0) {
+              return (
+                <p>This item has already been added to your shopping list.</p> // if we can add the actual item name {data.name}
```

Comment:
Do you mean you want to do something like `{this.state.name}` in place of "This item"? Or is this a stretch goal?

Reply:
Yes that's what we wanted to do. We moved this component up and deleted this file. So now we have access to this.state.name